contracts/src/v0.8/ccip/rmn/RMNRemote.sol

```diff
@@ -45,11 +45,10 @@ contract RMNRemote is OwnerIsCreator, ITypeAndVersion, IRMNRemote {
   }
 
   /// @dev the contract config
-  /// @dev note: minSigners can be set to 0 to disable verification for chains without RMN support
   struct Config {
     bytes32 rmnHomeContractConfigDigest; // Digest of the RMNHome contract config
-    Signer[] signers; // List of signers
-    uint64 minSigners; // Threshold for the number of signers required to verify a report
+    Signer[] signers; //                    List of signers
+    uint64 f; //                            Max number of faulty RMN nodes; f+1 signers are required to verify a report, must configure 2f+1 signers in total
```

Comment:
I believe this requires changes in chainlink-ccip cc @0xnogo

Reply:
Fault tolerance is usually number of NOPs = 3f+1. In the plugin code we require 2f+1 of those observations for consensus to pass. This comment suggest the offchain code handles this differently?